docs/api.md

```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
+|     \_id     | String |                               Unique universal identifier of this event action.                                |
+|     name     | Object |                             Human-translatable-readable name for the event action.                             |
+|     type     | String |                 Human readable type of a event that the action can be triggered to take place.                 |
+| abbreviation | Object |                   Human-translatable-readable abbreviation for the name of the event action.                   |
+| description  | Object |                         Human-translatable-readable description for the event action.                          |
+|     code     | String |                             Unique human-readable given code of this event action.                             |
+|    color     | String |               A color code(in hexadecimal format) used to differentiate event actions visually.                |
+|   function   | String |          Human-translatable-readable name of event function to which the action is categorized into.           |
+|  createdAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was created. |
+|  updatedAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was updated. |
+
+### Create EventAction
+
+To create a new event action, send a `POST` request to `https://api.ewea.io/v1/predefines/eventactions`. The following attributes are supported:
+
+<br/>
+
+|    Name     |  Type  |                                         Description                                         | Required |
+| :---------: | :----: | :-----------------------------------------------------------------------------------------: | :------: |
+|    code     | String |                   Unique human-readable given code of this event action.                    |  false   |
+|    type     | String |       Human readable type of a event that the action can be triggered to take place.        |   true   |
+|    name     | Object |                   Human-translatable-readable name for the event action.                    |   true   |
+| description | Object |                Human-translatable-readable description for the event action.                |  false   |
+|    color    | String |      A color code(in hexadecimal format) used to differentiate event action visually.       |  false   |
+|  function   | String | Human-translatable-readable name of event function to which the action is categorized into. |   true   |
+
+> Example Request
+
+```curl
+curl --request POST \
+--url https://api.ewea.io/v1/predefines/eventactions \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <apiKey>' \
+--data '{
+"name": { "en": "Working with the Regional Disaster Management Committee to
```

Comment:
Please use: Disseminating warning information to the public

Reply:
Done